### ISSUE TITLE
CI: restore CI failure Slack notifications 

### ISSUE THREAD

**Issue by fingolfin** [gap-system/gap#4390]
With Travis we used to get notifications for CI failures on Slack. We need this again, so that we notice when master or stable jobs start failing.

In order for this to maximize usefulness and minimize annoyance, in the past we configured this to only ping us on "edges": I.e. there was a notification when the build started failing; and one when it stopped failing. I guess for master/stable-* branches, it would also be OK to get a notification for each failure. But we do *not* want a notification for each success.

I suggest using a new Slack channel `#status` for this.

**Comment by wilfwilson** [comment]
@fingolfin has now renamed the `#travis` channel on Slack to `#status`.

**Comment by fingolfin** [comment]
There are tons of Slack notification actions; unfortunately it is difficult to decide which to use; there is no "official" one AFAICT; and I am not sure those I found are "trustworthy "... ah well.

more annoyingly, it is also unclear to me how to make it only notify on status *changes*... i didn't find anything on that yet. I guess we'd have to record (store) the state of the last run of a job somewhere... hrm... perhaps one can (ab)use the CI status indicator badge that we embed into  README.md?
